.github/workflows/ci.yml

```
@@ -33,3 +33,14 @@ jobs:
 
       - name: Unit test
         run: npm run test
+        
+      - name: Test Coverage
+        run: npm run test:coverage
```

```suggestion
      - name: Unit Test with Coverage
        run: npm run test:coverage
```

기존 `npm run test`는 삭제해도 됩니다! test:coverage가 coverage 파일 생성을 위해 유닛테스트를 모두 돌려줘서요 ㅎㅎ

---

아앗 맨위에 1번 부분을 빼먹었네요 ㅠㅠ